src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```diff
@@ -1,5 +1,5 @@
 services:
-    _defaults:
+      _defaults:
```

Comment:
🤔

Reply:
(I wanted to break the CI :p But YamlLint seems to be hard than `php bin/console yaml:lint`).